docs/install/ansible.rst

```diff
@@ -270,8 +288,19 @@ The Ansible playbooks are located in the ``ansible/`` directory:
 - ``utils.yml``: install extra system packages useful for debugging and system administration
 - ``users.yml``: create the tests users on the host
 - ``quotas.yml``: enable quotas on the host to limit disk usage
+- ``include-groups.yml``: add user groups to JupyterHub
 - ``cockpit.yml``: install Cockpit on the host as a monitoring tool
 - ``tljh.yml``: install TLJH and the Plasma TLJH plugin
 - ``https.yml``: enable HTTPS for TLJH
 - ``uninstall.yml``: uninstall TLJH only
 - ``site.yml``: the main playbook that references some of the other playbooks
+
+
+Running playbook on a given server
+----------------------------------
+
+If you have multiple servers defined in the ``hosts`` file, you can run a playbook on a single server with the ``--limit`` option:
+
+.. code-block:: bash
+
+    ansible-playbook site.yml -i hosts -u ubuntu --limit server1
```

Comment:
If we use `-i` for the inventory, should we use `-l` instead of `--limit`?

Reply:
You're perfectly right. I found sometime the character `l` a bit confusing. That's why I opted for the long form.

Reply:
Maybe we should keep `--limit` then? Or follow what the Ansible documentation uses?

Reply:
Ansible documentation tends to use `--limit`:
https://docs.ansible.com/ansible/latest/user_guide/intro_patterns.html
> You can also limit the hosts you target on a particular run with the --limit flag:

Reply:
ok, then let's keep it like it is. Thanks for checking!